crates/catalog/hms/src/catalog.rs

```diff
@@ -95,9 +106,156 @@ impl HmsCatalog {
             client: HmsClient(client),
         })
     }
+
+    /// Create and extract properties from `hive_metastore::hms::Database`.
+    pub fn properties_from_database(database: &Database) -> HashMap<String, String> {
```

Comment:
nit: Maybe we can extract those helper utils in another file to make `catalog.rs` clean?

Reply:
+1, also I'm thinking about putting all these properties such as "comment", "location" in to constants.

Reply:
I had them in utils before - and was not sure if they should stay there. I moved them back to utils - however I think the conversion `from_namespace` and `to_namespace` need an extra type like @liurenjie1024 suggested. This would clean up both the catalog and utils, i think.